docs/specifications/broker-authentication.md

```diff
@@ -0,0 +1,109 @@
+# Broker Authentication
```

Comment:
One last comment the LGTM.  Can you rename the directory to specs instead of specifications? A lot of projects I've seen used the shortened name.